pupil_src/capture/run_capture.bat

```diff
@@ -1,2 +1,2 @@
-PATH %PATH%..\..\pupil_external
+SET PATH=%PATH%;%~dp0..\..\pupil_external
```

Comment:
You can see this by running the batch script in a cmd and after closing do

    echo %PATH%

After this close the cmd and open a new one and do 

    echo %PATH%

You will see that ..\..\pupil_external will be removed.